examples/guided.py

```diff
@@ -205,21 +269,35 @@ def ask_user_questions():
 		archinstall.arguments['kernels'] = archinstall.select_kernel(kernels)
 
 	# Additional packages (with some light weight error handling for invalid package names)
-	print("Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed.")
-	print("If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt.")
+	print(
+		"Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed."
+	)
+	print(
+		"If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt."
+	)
 	while True:
 		if not archinstall.arguments.get('packages', None):
-			archinstall.arguments['packages'] = [package for package in input('Write additional packages to install (space separated, leave blank to skip): ').split(' ') if len(package)]
+			archinstall.arguments['packages'] = [
+				package
+				for package in input(
+					'Write additional packages to install (space separated, leave blank to skip): '
+				).split(' ')
+				if len(package)
+			]
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Weird allowed meta that creates unreadable conditions. If we don'e like the original syntax, we shouldn't re-write it like this, we should re-write the list creation outside of the variable definition if anything. Just because something is allowed doesn't make it good. Interesting concept tho.